operator/pom.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.streamshub</groupId>
+        <artifactId>console</artifactId>
+        <version>0.1.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>console-operator</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <!-- Container image properties for systemtests -->
+        <container-image.registry>localhost</container-image.registry>
+        <container-image.group>streamshub</container-image.group>
+        <container-image.tag>${project.version}</container-image.tag>
+        <container-image.push>false</container-image.push>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus.platform</groupId>
+                <artifactId>quarkus-operator-sdk-bom</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.streamshub</groupId>
+            <artifactId>console-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkiverse.operatorsdk</groupId>
+            <artifactId>quarkus-operator-sdk</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>crd-generator-apt</artifactId>
```

Comment:
You shouldn't need this dependency

Reply:
Thanks. Replaced `crd-generator-apt` with fabric8:generator-annotations and sundrio:builder-annotations.

Reply:
Hmm maybe I missed something: why were you using this dependency? QOSDK should generate the CRD for you automatically without needing to run the annotation processor…

Reply:
> Hmm maybe I missed something: why were you using this dependency? QOSDK should generate the CRD for you automatically without needing to run the annotation processor…

I must have picked it up from an example or older blog post, then came to rely on it for the transitive deps fabric8:generator-annotations and sundrio:builder-annotations, which are now direct.